### DISCUSSION TITLE
Emit on Publish / Save Event

### DISCUSSION THREAD

**Discussion by dainemawer** [avitorio/outstatic#91] [Ideas]
I've been using Outstatic for a while now, really love the platform. It's clean/minimal and precisely what I need. That said, it would be great if there were a hook to fire off post requests on publish or save. For instance, posting content to other platforms where my articles are syndicated without republishing them manually would be great. Is this currently possible or potentially possible?

**Comment by avitorio** [comment]
Hey @dainemawer, thanks for reaching out. I'm glad to hear you're enjoying your experience with Outstatic!

For the moment Outstatic doesn't provide any hooks for external actions... but I think this is possible. I just haven't given it much thought. Do you have any suggestions or ideas on how such a feature would look like? I don't mean technically, but even a quick step-by-step of the workflow you'd like to see in action can be helpful.

One of the things on the roadmap is to provide a `/plugins` folder within `/oustatic` to allow users to extend their usage of the framework. Hopefully it will be easy to create such functionality once that's in place.

**Reply by thecodedrift** [reply]
As an intermediate solution, would an RSS feed coupled with something like https://ifttt.com/ work? For more complex integrations this probably wouldn't work, but for the post + syndicate scenario, there's a lot of great tooling around RSS feeds we could use. Plus, then outstatic blogs would get RSS. 😄

**Reply by dainemawer** [reply]
Yeah, thats a great idea @jakobo - could definitely work with that solution. 
@avitorio - my original thought was perhaps a simple webhook that you could simply pass a callback to? Maybe even an API end point? Most headless CMS' seem to go with Webhooks so Im assuming there's a good reason for that.

**Reply by avitorio** [reply]
I think we'll eventually allow for different webhooks based on CMS actions. If you have any examples of cool implementations on other CMS's please let me know.

Meanwhile, you can probably add an rss feed to your site by [following this tutorial](https://blog.logrocket.com/adding-rss-feed-next-js-app/). I'd be interested to learn if this works.
